cookbooks/fb_storage/libraries/format_devices_provider.rb

```diff
@@ -238,7 +239,7 @@ def get_primary_work(storage)
           partitions.each do |p|
             storage.arrays.each do |array, info|
               if (info['members'].include?(p) ||
-                  (info['journal'] && info['journal'].include?(p))) &&
+                  (info['journal']&.include?(p))) &&
```

Comment:
@NaomiReeves, will this work fine with our current Chef version?

Reply:
I used it internally a bunch before i left, so I would imagine?